kong/cmd/utils/log.lua

```diff
@@ -24,6 +25,15 @@ function _M.set_lvl(lvl)
   end
 end
 
+function _M.disable()
+  old_lvl = log_lvl
```

Comment:
not that important, so let's drop this. But if we/I ever get bitten by this, you'll owe me a :beer:

😄